evennia/objects/objects.py

```diff
@@ -672,6 +672,55 @@ def for_contents(self, func, exclude=None, **kwargs):
         for obj in contents:
             func(obj, **kwargs)
 
+    def receive_action_msg(self, from_obj, message, mapping, raise_funcparse_errors=False, **kwargs):
+        """
+        Receive and process an action message from another object. Refer to msg_contents for the docs.
```

Comment:
The term `action message` means nothing in context. Better to describe it as a message that may or may not contain $You markup. 
This method also needs a full doc string to describe the input arguments.